From: Stefan Sundin <stefan@stefansundin.com>
Date: Fri, 12 Sept 2013 
Subject: Add build flags

Forwarded: no

Index: veracrypt/src/Makefile
===================================================================
--- veracrypt.orig/src/Makefile
+++ veracrypt/src/Makefile
@@ -54,6 +54,10 @@ export WXCONFIG_CFLAGS :=
 export WXCONFIG_CXXFLAGS :=
 WX_ROOT ?= ..
 
+CPPFLAGS+=$(shell dpkg-buildflags --get CPPFLAGS)
+CFLAGS+=$(shell dpkg-buildflags --get CFLAGS) $(CPPFLAGS) -Wno-sequence-point
+CXXFLAGS+=$(shell dpkg-buildflags --get CXXFLAGS) $(CPPFLAGS) -Wno-narrowing
+LFLAGS+=$(shell dpkg-buildflags --get LDFLAGS)
 
 export TC_BUILD_CONFIG := Release
 
Index: veracrypt/src/Build/Include/Makefile.inc
===================================================================
--- veracrypt.orig/src/Build/Include/Makefile.inc
+++ veracrypt/src/Build/Include/Makefile.inc
@@ -14,7 +14,7 @@ $(NAME): $(NAME).a
 
 clean:
 	@echo Cleaning $(NAME)
-	rm -f $(APPNAME) $(NAME).a $(OBJS) $(OBJSEX) $(OBJS:.o=.d) *.gch
+	rm -f $(APPNAME) $(NAME).a $(OBJS) $(OBJS:.o=.d) *.gch $(RESOURCES) SystemPrecompiled.d
 
 %.o: %.c
 	@echo Compiling $(<F)
