From bd843ffa05147aab80e38ee668b0556302d2bfc9 Mon Sep 17 00:00:00 2001
From: Mounir IDRASSI <mounir.idrassi@idrix.fr>
Date: Mon, 2 Oct 2023 18:02:33 +0200
Subject: [PATCH] Linux: Fix compilation error on some 32-bit machines.

---
 src/Common/PCSCException.cpp | 7 +++++--
 1 file changed, 5 insertions(+), 2 deletions(-)

diff --git a/src/Common/PCSCException.cpp b/src/Common/PCSCException.cpp
index 93fff18187..dc5eead323 100644
--- a/src/Common/PCSCException.cpp
+++ b/src/Common/PCSCException.cpp
@@ -140,14 +140,17 @@ namespace VeraCrypt
 	{
 		Exception::Deserialize(stream);
 		Serializer sr(stream);
-		sr.Deserialize("ErrorCode", ErrorCode);
+		int64 v;
+		sr.Deserialize("ErrorCode", v);
+		ErrorCode = (LONG_PCSC)v;
 	}
 
 	void PCSCException::Serialize(shared_ptr <Stream> stream) const
 	{
 		Exception::Serialize(stream);
 		Serializer sr(stream);
-		sr.Serialize("ErrorCode", ErrorCode);
+		int64 v = (int64)ErrorCode;
+		sr.Serialize("ErrorCode", v);
 	}
 
 #	define TC_EXCEPTION(TYPE) TC_SERIALIZER_FACTORY_ADD(TYPE)
